docs/MainSite/Main.md

```diff
@@ -74,8 +75,7 @@ eXtreme Spacetimes (SXS) research program into the multi-messenger
 astrophysics of neutron star mergers, core-collapse supernovae, and
 gamma-ray bursts.
```

Comment:
No love for black holes??

Reply:
That is outside the scope of this PR

Reply:
If you change this sentence in the contributing guidelines, I suggest that for consistency you may as well change it here, too. For example:

> SpECTRE is being developed in support of our collaborative Simulating eXtreme Spacetimes (SXS) research program into the multi-messenger astrophysics of black-hole and neutron-star mergers, core-collapse supernovae, and gamma-ray bursts.